mkdocs.yml

```diff
@@ -190,7 +189,14 @@ nav:
                       - microservices/device/services/device-onvif-camera/supplementary-info/getting-started-with-docker-security.md
                       - microservices/device/services/device-onvif-camera/supplementary-info/ws-discovery.md
                       - microservices/device/services/device-onvif-camera/supplementary-info/onvif-footnotes.md
-          - Device REST: microservices/device/services/device-rest.md
+          - Device REST:
+              - Purpose: microservices/device/services/DeviceRest/Purpose.md
+              - Getting Started: microservices/device/services/DeviceRest/GettingStarted.md
+              - How-To Guides:
+                  - Testing: microservices/device/services/DeviceRest/howto/Testing.md
+              - Configuration: microservices/device/services/DeviceRest/Configuration.md
+              - API Reference: microservices/device/services/DeviceRest/ApiReference.md
+              - Source Code: microservices/device/services/DeviceRest/SourceCode.md
```

Comment:
We should add Additional Details below Source Code tab.
Additional Details:
   - Device Commands: microservices/device/services/DeviceModbus/details/DeviceCommands.md
   - Device Definitions: microservices/device/details/DeviceDefinitions.md
   - Device Profiles: microservices/device/details/DeviceProfiles.md
  
Or you will add it later once comlete all of them?